serrano/resources/async/preview.py

```diff
@@ -0,0 +1,62 @@
+from django.conf.urls import patterns, url
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
+from django_rq import enqueue
+
+from serrano.resources.preview import PreviewResource
+
+
+class AsyncPreviewResource(PreviewResource):
+    """
+    Resource for asynchronously retrieving a data preview before exporting.
+
+    This method does not return the data itself. Instead, a "see-other"
+    response is returned with the ID of the job that is generating the data.
+    The client can use this job ID to poll the job status and retrieve the
+    result when the job is complete.
+
+    Data is formatted using a JSON+HTML exporter which prefers HTML formatted
+    or plain strings. Browser-based clients can consume the JSON and render
+    the HTML for previewing.
+    """
+
+    URL_PATTERN_NAME = 'serrano:async:preview'
+
+    def _get_results(self, request, **kwargs):
+        # TODO: Should we store this in the DB or just let this job result
+        # live in memory and let Redis clear it as needed?
```

Comment:
Let's store it in Redis since it's just a blob of data. That was the approach I took for htq as well.